client-v2/src/components/FrontsEdit/CollectionComponents/Collection.tsx

```diff
@@ -87,16 +98,17 @@ const createMapStateToProps = () => {
     }),
     displayEditWarning: editWarningSelector(selectSharedState(state), {
       collectionId: id
-    })
+    }),
+    isOpen: selectIsCollectionOpen(state, id)
   });
 };
 
-const mapDispatchToProps = (dispatch: Dispatch) => {
-  return {
-    publishCollection: (id: string, frontId: string) =>
-      dispatch(publishCollection(id, frontId))
-  };
-};
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  publishCollection: (id: string, frontId: string) =>
+    dispatch(publishCollection(id, frontId)),
+  onChangeOpenState: (id: string, isOpen: boolean) =>
+    dispatch(isOpen ? editorOpenCollections(id) : editorCloseCollections(id))
```

Comment:
Bit of a funny one, but the API matches the fronts UI and is quite useful elsewhere!